src/python/pants/backend/python/tasks2/pytest_run.py

```diff
@@ -209,6 +209,14 @@ def _execute(self, all_targets):
   class InvalidShardSpecification(TaskError):
     """Indicates an invalid `--test-shard` option."""
 
+  class CoverageConfigFileInvalid(TaskError):
+    """Indicates that the supplied configuration file is unparseable
+    """
```

Comment:
Small nit - please pull up `"""` to prior line since it all fits in <100 cols. Also end the summary sentence with a period.
Here and below in `CoverageConfigFileUnreadable` too.